### PR TITLE
DDF-3535 SAML Session viewing

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Adds command to list all sessions with their active SPs

This is related to #2819 but only provides the session-listing capabilities, not the deletion/logout capabilities.

#### Who is reviewing it? 
@brjeter @Lambeaux 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)

#### Select relevant component teams: 
@codice/security 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@stustison

#### How should this be tested? (List steps with links to updated documentation)
CI build and then a test of the list command. As this is a subset of the tested functionality already merged to master, a sanity test should be sufficient.

#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
[DDF-3535](https://codice.atlassian.net/browse/DDF-3535)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: platform/security/command/security-command-sessionmanager/src/main/java/org/codice/ddf/security/command/sessionmanager/ListSessionsCommand.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@Service
+@Command(scope = "security", name = "listsessions", description = "Lists all active SAML sessions.")
+public class ListSessionsCommand implements Action {
+
+  @Reference private SessionHandler sessionHandler;
+
+  @SuppressWarnings("squid:S106" /* System.out for Karaf console */)
+  @Override
+  public Object execute() throws Exception {
+    final Map<String, Set<String>> activeSessions = sessionHandler.getActiveSessions();
```
Comment (stustison) [inline platform/security/command/security-command-sessionmanager/src/main/java/org/codice/ddf/security/command/sessionmanager/ListSessionsCommand.java]: Are we sure this doesn't blow up with an NPE if the IdP is uninstalled?

Comment (coyotesqrl) [reply platform/security/command/security-command-sessionmanager/src/main/java/org/codice/ddf/security/command/sessionmanager/ListSessionsCommand.java]: I'll confirm.

Comment (coyotesqrl) [reply platform/security/command/security-command-sessionmanager/src/main/java/org/codice/ddf/security/command/sessionmanager/ListSessionsCommand.java]: Better than I'd expected, actually. Karaf simply unregisters the command if IdP server is stopped and re-registers it when the IdP is restarted.

**Pull request merged**